rdtools/analysis_chains.py

```diff
@@ -204,7 +204,9 @@ def _calc_clearsky_poa(self, times=None, rescale=True, **kwargs):
         Parameters
         ----------
         times : pandas.DateTimeIndex
-            times on for which to calculate clearsky poa
+            times on for which to calculate clearsky poa.  If not provided then
+            it will be simulated at 1-minute frequency and averaged to match the
+            index of self.poa_global
```

Comment:
Something to think about:  back in `2.0.0-alpha.0` this was a public method, but it's a private method now and the `times` and `rescale` parameters never get used anywhere, so consider removing them